samples/newChat/js/message.js

```diff
@@ -0,0 +1,394 @@
+'use strict';
+
+function Message() {
+    this.container = null;
+    this.attachmentPreviewContainer = null;
+    this.limit = appConfig.messagesPerRequest || 50;
+
+    this.dialogTitle = null;
+    this._typingTimer = null;
+    this._typingTime = null;
+    this.typingUsers = {};
+}
+
+Message.prototype.init = function(){
+    var self = this;
+
+    self.container = document.querySelector('.j-messages');
+    self.attachmentPreviewContainer = document.querySelector('.j-attachments_preview');
+    self.dialogTitle = document.querySelector('.j-content__title');
+
+    document.forms.send_message.addEventListener('submit', function(e){
+        e.preventDefault();
```

Comment:
If you want to use event.stopPropagation(), you also need to add `return false` (FF will reload page)
We need to remove default browser action, that's why we use preventDefault method

https://developer.mozilla.org/ru/docs/Web/API/Event/stopPropagation
https://developer.mozilla.org/ru/docs/Web/API/Event/preventDefault